share/spice/brainy_quote/brainy_quote.js

```diff
@@ -1,32 +1,76 @@
 (function(env) {
     env.ddg_spice_brainy_quote = function(api_result) {
 
+        Handlebars.registerHelper("getAuthor", function (arg1, arg2) {
+            if (arg2 === undefined) {
+                return;
+            }
+            return arg1 === undefined ? arg2.replace(/ quote$/, "") : arg1;
```

Comment:
this could actually be further simplified: 

`return arg1 || arg2.replace(/ quote$/, "")`